src/main/java/org/folio/models/claiming/IntegrationDetailField.java

```diff
@@ -6,6 +6,7 @@
 @Getter
 @AllArgsConstructor
 public enum IntegrationDetailField {
+  TENANT("tenant"),
```

Comment:
Do you think this field is needed?

Reply:
> Do you think this field is needed?

Yeah I think so, just in case there may be some other integrations details with different tenants getting used to create jobs: 

![image](https://github.com/user-attachments/assets/18716d06-3d62-4526-97c8-429e9d272de8)

Need some end-to-end testing to verify this behaviour though